### ISSUE TITLE
Setup:upgrade fail with sample data

### ISSUE THREAD

**Issue by rafaelstz** [netz98/n98-magerun2#500]
When I've installed a Magento 2.3 + Sample data and try to run `n98-magerun.phar setup:upgrade` I got this issue below. If I run via `bin/magento` it works.

```
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'm23.catalog_category_product_index_store1_tmp' doesn't exist, query was: DESCRIBE `catalog_category_product_index_store1_tmp`
```
<img width="1149" alt="Screen Shot 2019-11-21 at 7 35 17 PM" src="https://user-images.githubusercontent.com/610598/69384159-4deb8900-0c9a-11ea-99f1-fba7b8856445.png">

**Comment by dzschille** [comment]
I had this error message today while using the "fast simple import". Reason was not enough RAM for MySQL. Add more RAM and try again.

**Comment by rafaelstz** [comment]
@dzschille I ran it with `php -d memory_limit=-1`

**Comment by dzschille** [comment]
My maschine (VM) hadn't enough RAM so i had to add it there. Memory limit also counts for PHP but in this case the MySQL server hadn't enough memory.

**Comment by cmuench** [comment]
Should be fixed in v7.0.0.
Please re-open if issue still exists.
You can test with development version.
